Controller/ClientController.php

```diff
@@ -25,28 +29,46 @@ class ClientController extends BaseController
     /**
      * @Route("/", name="client_show", methods={"GET","POST"})
      */
-    public function show(Request $request, Process $process): Response
+    public function show(Request $request, Process $process, CprServiceInterface $cprService, LoggerInterface $logger): Response
     {
         $client = $process->getClient();
 
         $changeProcessStatusForm = $this->createChangeProcessStatusForm($process);
         $this->handleChangeProcessStatusForm($request, $changeProcessStatusForm);
 
+        $cpr = str_replace('-', '', $process->getClientCPR());
```

Comment:
This `str_replace` expression occurs (at least) three times in this PR. Consider letting the `CprService` provide a public  `normalizeCpr` method or let the service internally normalize before calling the actual api.